app/vmalert/group.go

```diff
@@ -16,7 +16,9 @@ import (
 // Group is an entity for grouping rules
 type Group struct {
 	Name  string
+	Interval time.Duration `yaml:"interval"`
```

Comment:
please populate this field during the parsing and use only it in downstream code.
logic if it's set do nothing otherwise use global setting

Reply:
Fixed